sdk/resourcemanager/ci.mgmt.yml

```diff
@@ -307,6 +308,7 @@ pr:
     - sdk/servicefabric/Azure.ResourceManager.ServiceFabric
     - sdk/servicefabricmanagedclusters/Azure.ResourceManager.ServiceFabricManagedClusters
     - sdk/servicelinker/Azure.ResourceManager.ServiceLinker
+    - sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking.TrafficController
```

Comment:
This should be  sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking in my opinion
We are not creating a sdk for the resource but entire Service Networking Name space